ProcessLib/ThermoMechanics/ThermoMechanicsProcess.cpp

```diff
@@ -201,6 +281,28 @@ void ThermoMechanicsProcess<DisplacementDim>::initializeConcreteProcess(
     }
 }
 
+template <int DisplacementDim>
+void ThermoMechanicsProcess<DisplacementDim>::initializeBoundaryConditions()
+{
+    if (_use_monolithic_scheme)
+    {
+        const int process_id_of_thermomechancs = 0;
```

Comment:
This value was already stored; see the create process code, where the decision monolithic/staggered is made...

Reply:
Yes, you are right. For monolithic scheme, the process ID is 0, and the stored values are that  mechanics_process_id =0 and  heat_conduction_process_id =0. For this case, using either  mechanics_process_id =0 or  heat_conduction_process_id =0 would be misleading. Therefore this local constant is declared.

Reply:
```suggestion
        const int process_id_of_thermomechanics = 0;
```

Reply:
Corrected.